ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -32,21 +34,76 @@ def __init__(self):
         rospy.Subscriber('/base_waypoints', Lane, self.waypoints_cb)
 
         # TODO: Add a subscriber for /traffic_waypoint and /obstacle_waypoint below
-
+        rospy.Subscriber('/traffic_waypoint', Int32, self.traffic_cb)
```

Comment:
I suspect there should be `Waypoint` instead of `Int32`, analogously to Lane from the previous line. But take into account that I don't know jack at this point.